server/src/main/java/org/opensearch/tasks/consumer/TopNSearchTasksLogger.java

```diff
@@ -97,4 +104,12 @@ private void logTopResourceConsumingQueries() {
             SEARCH_TASK_DETAILS_LOGGER.info(new SearchShardTaskDetailsLogMessage(topQuery.v2()));
         }
     }
+
+    void setLogTopQueriesSize(int topQueriesSize) {
+        this.topQueriesSize = topQueriesSize;
+    }
+
+    void setTopQueriesLogFrequencyInNanos(TimeValue timeValue) {
```

Comment:
Please make this method `private`.

Reply:
Will need this to be package-private since we need to modify frequency for testing, otherwise UTs will take 60+ seconds to finish. Code reference: https://github.com/opensearch-project/OpenSearch/pull/6716/files#diff-7f1abfc2317c2c29386a05b19737625a551e7f89d9c1da1ad81065e2bb61bf17R76